src/main/java/org/jabref/logic/openoffice/OpenOfficeFileSearch.java

```diff
@@ -30,30 +29,35 @@ public class OpenOfficeFileSearch {
     public static List<Path> detectInstallations() {
         if (OS.WINDOWS) {
             List<Path> programDirs = findWindowsOpenOfficeDirs();
-            return programDirs.stream().filter(dir -> FileUtil.find(OpenOfficePreferences.WINDOWS_EXECUTABLE, dir).isPresent()).collect(Collectors.toList());
+            return programDirs.stream().filter(dir -> FileUtil.find(OpenOfficePreferences.WINDOWS_EXECUTABLE, dir).isPresent()).toList();
         } else if (OS.OS_X) {
             List<Path> programDirs = findOSXOpenOfficeDirs();
-            return programDirs.stream().filter(dir -> FileUtil.find(OpenOfficePreferences.OSX_EXECUTABLE, dir).isPresent()).collect(Collectors.toList());
+            return programDirs.stream().filter(dir -> FileUtil.find(OpenOfficePreferences.OSX_EXECUTABLE, dir).isPresent()).toList();
         } else if (OS.LINUX) {
             List<Path> programDirs = findLinuxOpenOfficeDirs();
-            return programDirs.stream().filter(dir -> FileUtil.find(OpenOfficePreferences.LINUX_EXECUTABLE, dir).isPresent()).collect(Collectors.toList());
+            return programDirs.stream().filter(dir -> FileUtil.find(OpenOfficePreferences.LINUX_EXECUTABLE, dir).isPresent()).toList();
+        } else {
+            return List.of();
         }
-        return new ArrayList<>(0);
     }
 
     private static List<Path> findOpenOfficeDirectories(List<Path> programDirectories) {
         BiPredicate<Path, BasicFileAttributes> filePredicate = (path, attr) ->
                 attr.isDirectory() && (path.toString().toLowerCase(Locale.ROOT).contains("openoffice")
                         || path.toString().toLowerCase(Locale.ROOT).contains("libreoffice"));
 
-        return programDirectories.stream().flatMap(dirs -> {
-            try {
-                return Files.find(dirs, 1, filePredicate);
-            } catch (IOException e) {
-                LOGGER.error("Problem searching for openoffice/libreoffice install directory", e);
-                return Stream.empty();
-            }
-        }).collect(Collectors.toList());
+        return programDirectories.stream()
+                                 .flatMap(dirs -> {
+                                     try {
+                                         return Files.find(dirs, 1, filePredicate);
+                                     } catch (IOException e) {
+                                         LOGGER.error("Problem searching for openoffice/libreoffice install directory", e);
+                                         return Stream.empty();
+                                     }
+                                 })
+                                 // On Windows, the executable is nested in subdirectory "program"
+                                 .map(dir -> dir.resolve("program"))
```

Comment:
Maybe add a check for Windows there? Not sure under linux (need to check on mac later this evening but there it seems to work out of the box)

Reply:
Damn, I was sure, it was called only at the Windos case